src/patterns/components/pagination/collection.yaml

```diff
@@ -4,8 +4,8 @@ description: |
   It serves as navigation and informs users where they currently are and where they
   can go in the future.
 information:
-  - Spark-Core Pagination JS will take care of adding in the correct aria-label
-    to each link on first page load and when the items are tapped or clicked.
+  - As a general guideline, you should use the "default" variant when the pager has 3 or fewer pages,
```

Comment:
@bobwise  can you make a quick update in your current branch, 3 should be three here.